src/pages/iou/request/step/IOURequestStepScan/index.tsx

```diff
@@ -321,8 +324,14 @@ function IOURequestStepScan({
             // If the transaction was created from the global create, the person needs to select participants, so take them there.
             // If the user started this flow using the Create expense option (combined submit/track flow), they should be redirected to the participants page.
             // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-            if ((transaction?.isFromGlobalCreate && iouType !== CONST.IOU.TYPE.TRACK && !report?.reportID) || iouType === CONST.IOU.TYPE.CREATE) {
```

Comment:
Similar question, please elaborate on the purpose of this changes.

Reply:
The `iouType` for global create is `create` and the logical-or will check true on global create, so nothing different from this changes.